src/frontend/efiling-frontend/src/components/page/rush/Rush.js

```diff
@@ -105,7 +106,45 @@ export default function Rush({ payment }) {
   const [countries, setCountries] = useState([]);
   const [continueBtnEnabled, setContinueBtnEnabled] = useState(false);
 
-  const handleContinue = () => {};
+  const handleContinue = () => {
+    const formData = new FormData();
+    for (let i = 0; i < files.length; i += 1) {
+      formData.append("files", files[i]);
+    }
+    const req = {
+      rushType: fields.rushType,
+      firstName: fields.firstName,
+      lastName: fields.surname,
+      organization: fields.org,
+      phoneNumber: fields.phoneNumber,
+      email: fields.email,
+      country: fields.country,
```

Comment:
During the phone number validation fix, I switched country from being a string to being an object with code and description fields. Does this need to be switched to be `fields.country.description` or does the rush submission work with the object?